README.md

```diff
@@ -11,13 +11,14 @@ Folks notice minor and larger issues with the Infra Standard all the time and we
 your help fixing those. Pull requests for typographical and grammar errors are also most welcome.
 
 We'd be happy to mentor you through this process. If you're interested and need help getting
-started, leave a comment on the issue or ask around [on IRC](https://wiki.whatwg.org/wiki/IRC).
+started, review [WHATWG contributing guidelines](https://github.com/whatwg/meta/blob/master/CONTRIBUTING.md),
```

Comment:
Missing "the" before the link.

Reply:
Fixed in https://github.com/whatwg/infra/pull/170/commits/2ecd75666a6b6fb485d65b0890f52f4ad7657228